c_reference/tests/utils/test_quantized_utils.c

```diff
@@ -18,6 +18,15 @@ int check_output(const INT_T* const pred, const INT_T* const expected,
   return 0;
 }
 
+// Test v_q_treesum() function.
+int test_v_q_treesum() {
+  INTM_T qvec_A[8] = {-425, -169, -3534, 524, -2739, 87, 52, 292};
+  const INTM_T expected[1] = {-738};
```

Comment:
can we try to input a full activation map corresponding to an image.

Reply:
Adding the convolution output of the third filter in MBConv (obtained from SeeDot's code), as a test.